elastalert/config.py

```diff
@@ -29,7 +29,8 @@
 rule_schema = jsonschema.Draft4Validator(yaml.load(open(os.path.join(os.path.dirname(__file__), 'schema.yaml'))))
 
 # Required global (config.yaml) and local (rule.yaml)  configuration options
-required_globals = frozenset(['run_every', 'rules_folder', 'es_host', 'es_port', 'writeback_index', 'buffer_time'])
+required_globals = frozenset(['run_every', 'rules_folder', 'es_host', 'es_port', 'writeback_index',
+                              'writeback_alias', 'buffer_time'])
```

Comment:
Can this be optional? I'd like to not introduce breaking changes. I think it's OK to create the index alias by default, but i'd also like to see ``writeback_alias`` have a default value, "elastalert_alerts" is probably ok.

Reply:
Yes of course, it would make a lot more sense to have `writeback_alias` as an optional value.